selenium_webdriver/src/main/java/pages/MainPage.java

```diff
@@ -0,0 +1,95 @@
+package pages;
+
+import models.User;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class MainPage extends BasePage {
+    static final String MENU_BUTTON_ID = "'react-burger-menu-btn'";
```

Comment:
why do you need static variables?

Reply:
In this particular place we do not because we are free of memory saving in general.
Fixed.